requirements/run.txt

```diff
@@ -1,4 +1,5 @@
 python
 ruamel.yaml
+docutils
```

Comment:
Wait, really? Is this truly a run requirement?

Reply:
https://github.com/regro/regolith/blob/cb94f4a8c5a5401aeaec655f5125354ada46c786/regolith/emailer.py

Reply:
Fair :)